commands/user_e2e_test.go

```diff
@@ -136,3 +136,64 @@ func (s *MmctlE2ETestSuite) TestSearchUserCmd() {
 		s.Equal("Unable to find user '"+emailArg+"'", printer.GetErrorLines()[0])
 	})
 }
+
+func (s *MmctlE2ETestSuite) TestDeleteAllUserCmd() {
+	s.SetupTestHelper().InitBasic()
+
+	s.Run("Delete all user as unpriviliged user", func() {
+		printer.Clean()
+
+		cmd := &cobra.Command{}
+		confirm := true
+		cmd.Flags().BoolVar(&confirm, "confirm", confirm, "confirm")
+
+		err := deleteAllUsersCmdF(s.th.Client, cmd, []string{})
+		s.Require().NotNil(err)
+		s.Len(printer.GetLines(), 0)
+		s.Len(printer.GetErrorLines(), 0)
+
+		// expect users not deleted
+		_, err = s.th.App.GetUser(s.th.BasicUser2.Id)
+		s.Require().Nil(err)
+	})
+
+	s.Run("Delete all user as system admin", func() {
+		printer.Clean()
+
+		cmd := &cobra.Command{}
+		confirm := true
+		cmd.Flags().BoolVar(&confirm, "confirm", confirm, "confirm")
+
+		err := deleteAllUsersCmdF(s.th.SystemAdminClient, cmd, []string{})
+		s.Require().NotNil(err)
+		s.Len(printer.GetLines(), 0)
+		s.Len(printer.GetErrorLines(), 0)
+
+		// expect users not deleted
+		_, err = s.th.App.GetUser(s.th.BasicUser2.Id)
+		s.Require().Nil(err)
+	})
+
+	s.Run("Delete all user", func() {
```

Comment:
We should add a direct reference to local mode here to clarify the test case:

```suggestion
	s.Run("Delete all users through local mode", func() {
```

Reply:
Agree